src/main/java/seedu/address/model/task/State.java

```diff
@@ -1,7 +1,7 @@
 package seedu.address.model.task;
 
 public enum State {
-    DONE("done"), PENDING("pending");
+    COMPLETE("complete"), INCOMPLETE("incomplete");
```

Comment:
May need to capitalise the first letter so that the UI is standerdized

Reply:
okie will change